guides/release-notes.md

```diff
@@ -8,6 +8,30 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 
 ---
 
+### Migration of frontend APIs
+
+In an effort to streamline our APIs we have migrated our frontend modules to contain the `-frontend` suffix. Legacy APIs were mostly divided between frontend and backend functionality and while the backend modules were indicated by a `-backend` suffix in the module name, the frontend modules had no indication.
+
+With the introduction of [universal modules](https://www.wix.com/velo/reference/api-overview/api-versions#api-overview_api-versions_universal-modules) we have now migrated the legacy frontend modules to clearly indicate that they are frontend only.
```

Comment:
Recommend tightening this up a bit. Not clear what "streamline our APIs" means - what's the benefit to the user here?